frontend/public/components/factory/Table/TableHeader.tsx

```diff
@@ -22,9 +22,10 @@ export const TableHeader: React.FCC<TableHeaderProps> = ({
           const sortable = sortField || sortFunc || sort;
           return (
             <Th
-              key={id}
+              key={id ?? title}
```

Comment:
FYI @TheRealJon, this is to fix a bug introduced in https://github.com/openshift/console/pull/14436.  If `id` is undefined, we get no `key`.